docs/create_broker.md

```diff
@@ -0,0 +1,246 @@
+retrieving and sanitizing data, and creating TOM alerts from it.
+
+Begin by defining the class, its name and default form. In our case, the name
+will simply be 'MyBroker', and the form will be `MyBrokerForm` - the form that we
+just defined!
+
+```python
+class MyBroker:
+    name = 'MyBroker'
+    form = MyBrokerForm
+```
+
+## Required Broker Class Methods
+Each TOM alert broker is required to have a base set of class methods. These
+methods enable the conversion of remote alert data into TOM-specific
+alerts and targets.
+
+#### `fetch_alerts` Class Method
+`fetch_alerts` is used to query the remote broker, and return a list
+of results depending on the parameters passed into the query, so that
+these results may be displayed on the query results page. In our case, `fetch_alerts`
+will only filter on name, but this can be easily extended to other query parameters.
+
+```python
+@classmethod
+def fetch_alerts(clazz, parameters):
+    response = requests.get(broker_url)
+    response.raise_for_status()
+    test_alerts = response.json()
+    return [alert for alert in test_alerts if alert['name'] == parameters['name']]
+```
+
+Our implementation will get a response from our test broker source, check that our
+request was successful, and return a list of all alerts whose name field matches the
+name passed into the query.
+
+#### `fetch_alert` Class Method
+`fetch_alert` is similar to `fetch_alerts`, but fetches a single alert with a given
+`alert_id`. Once a query has been performed, any resulting alert may be made into
+a TOM target directly from the query results page.
+
+```python
+@classmethod
+def fetch_alert(clazz, alert_id):
+    response = requests.get(broker_url)
+    test_alerts = response.json()
+    response.raise_for_status()
+    for alert in test_alerts:
+        if (alert['id'] == int(alert_id)):
+            return alert
+    return None
+```
+
+We query the broker, obtain our JSON data, and return the alert whose id
+matches the `alert_id` parameter, so a TOM Alert may be created from it.
+
+#### `to_generic_alert` Class Method
+In order to standardize alerts and display them in a consistent manner,
+the `GenericAlert` class has been defined within the `tom_alerts` library.
+This broker method converts a remote alert into a TOM Toolkit `GenericAlert`.
+
+```python
+@classmethod
+def to_generic_alert(clazz, alert):
+    return GenericAlert(
+        timestamp=alert['timestamp'],
+        url=broker_url,
+        id=alert['id'],
+        name=alert['name'],
+        ra=alert['ra'],
+        dec=alert['dec'],
+        mag=alert['mag'],
+        score=alert['score']
+    )
+```
+In our case, the `GenericAlert` attributes match up *almost* directly with our test
+data. How convenient! We'll just go ahead and define the `GenericAlert`'s `url`
+field as the `broker_url` we retrieved our test data from.
+
+```python
+@classmethod
+def to_generic_alert(clazz, alert):
```

Comment:
I think the above example to_generic_alert is sufficient, no need to repeat it here.